common/components/common/tags/TagSelectorCollapsible.jsx

```diff
@@ -100,13 +99,23 @@ class TagSelectorCollapsible extends React.Component<Props, State> {
               optionDisplay={tag => this._displayTag(tag)}
               optionEnabled={tag => this._tagEnabled(tag)}
               onOptionSelect={this.selectTag.bind(this)}
+              selectedByCategory={this._selectedByCategory()}
             />
             )
           : null
         }
       </div>
     );
   }
+  _selectedByCategory(): void {
```

Comment:
This generates selected item counts, which seems to work okay, and is what I planned to use to figure out if any given Select All checkbox should be checked, along the lines of "I'm the select all checkbox for the Data subcategory, Data has a length of 2 items, and this.props.selectedByCategory has data = 2, check the box"